.github/workflows/1_check_for_updates.yml

```diff
@@ -0,0 +1,40 @@
+name: Check for Prisma CLI Update
+
+on:
+  schedule:
+    - cron: '*/5 * * * *'
+
+env:  
+  PRISMA_TELEMETRY_INFORMATION: "language-tools check-for-update.yml"
+  GH_TOKEN: ${{ secrets.GH_TOKEN }}
+
+jobs:
+  check:
+    name: Check for Prisma CLI update
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Check for Prisma CLI update
+        id: check_update
+        run: npm run check_for_update
```

Comment:
Is there a benefit from running this via `npm run` vs. the script directly?

Reply:
no, not really. I replaced these to be called directly